ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -623,6 +623,27 @@ + (void)configureDocumentController:(PTFlutterDocumentController*)documentContro
                         [documentController setDefaultEraserType:defaultEraserType];
                     }
                 }
+                else if ([key isEqualToString:PTReflowOrientationKey]) {
+                    
+                    NSString *reflowOrientation = [PdftronFlutterPlugin getConfigValue:configPairs configKey:PTReflowOrientationKey class:[NSString class] error:&error];
+                    
+                    if (!error && reflowOrientation) {
+                        if ([PTReflowOrientationHorizontalKey isEqualToString:reflowOrientation]) {
+                            documentController.reflowViewController.scrollingDirection = PTReflowViewControllerScrollingDirectionHorizontal;
+                        } else {
```

Comment:
Would be better to check for equality to vertical key here, so that if someone passes a bogus string, it doesn't change it to vertical as a result (better would be no change).